### ISSUE TITLE
Incorrect loading of sheets in PhpOffice\PhpSpreadsheet\Reader\Xlsx::listWorksheetInfo()

### ISSUE THREAD

**Issue by Nimmer** [PHPOffice/PhpSpreadsheet#2142]
This is:

```
- [x] a bug report
- [x] **not** a usage question (ask them on https://stackoverflow.com/questions/tagged/phpspreadsheet or https://gitter.im/PHPOffice/PhpSpreadsheet)
```

### What is the expected behavior?
it should work

### What is the current behavior?
it does not work

### What are the steps to reproduce?
doing imports with the laravel excel package and its WithChunkReading (https://docs.laravel-excel.com/3.1/imports/chunk-reading.html)

i tried to pinpoint the problem here
[Link to github commit that broke it](https://github.com/PHPOffice/PhpSpreadsheet/commit/43d1da6fa0280c270938b1bf09943ff78458ed1b?_pjax=%23js-repo-pjax-container#commitcomment-51673597)

### Which versions of PhpSpreadsheet and PHP are affected?
1.7.1

**Comment by MarkBaker** [comment]
Do you mean 1.7.1 or 1.17.1

What you may want to do if you're concerned is test with [PR #2088](https://github.com/PHPOffice/PhpSpreadsheet/pull/2088) because that's the first phase of a fairly major rewrite of the Xlsx Reader changing the way that we handle namespacing and the Xml in general for Xslx

**Comment by Nimmer** [comment]
im so soryy, i definitely meant 1.17.1

**Comment by oleibman** [comment]
Closing. Fixed by PR #1895 which was part of 1.18.0 (May 31, 2021).
